rocksdb/backup.pxd

```diff
@@ -9,12 +9,17 @@ from .status cimport Status
 from .db cimport DB
 from .env cimport Env
 
+# TODO: For rocksdb >= 6.21.0, change to `rocksdb/utilities/backup_engine.h`.
 cdef extern from "rocksdb/utilities/backupable_db.h" namespace "rocksdb":
     ctypedef uint32_t BackupID
 
+    # TODO: For rocksdb >= 6.21.0, rename to `BackupEngineOptions`.
     cdef cppclass BackupableDBOptions:
         BackupableDBOptions(const string& backup_dir)
 
+    # TODO: For rocksdb >= 6.21.0, remove this type def.
+    ctypedef BackupableDBOptions BackupEngineOptions
```

Comment:
It seems this is not enough, now we have issues in the `cpp` file: https://github.com/NightTsarina/python-rocksdb/actions/runs/1960944886

I guess you need to move these compatibility definitions to the pure C++ fiiles?

Reply:
Probably not worth the complexity. I did this for another reason originally, when I thought it was more necessary. I just reverted to the old name for now. We can change everything to `BackupEngineOptions` when we drop compatibility with earlier versions. (I added a note.)